torch/csrc/jit/codegen/cuda/ir_base_nodes.cpp

```diff
@@ -90,9 +90,7 @@ Val::Val(IrBuilderPasskey passkey, ValType _vtype, DataType _dtype)
 Val::Val(const Val* src, IrCloner* ir_cloner)
     : Statement(src, ir_cloner),
       vtype_(src->vtype_),
-      dtype_(src->dtype_),
-      is_fusion_input_(src->is_fusion_input_),
-      is_fusion_output_(src->is_fusion_output_) {}
```

Comment:
Wow, nice catch. How could this have been working so far :astonished:

Reply:
I don't know, this was definitely on the scarier side of bugs.